scripts/audit_branches.py

```
@@ -0,0 +1,98 @@
+            check=False,  # We handle errors based on returncode or empty output
+            capture_output=True,
+            text=True
+        )
+        if result.returncode == 0:
+            return result.stdout.strip()
+        return None
+    except Exception as e:
+        print(f"Error executing command {' '.join(args)}: {e}", file=sys.stderr)
+        return None
+
+def main():
```

Ruff (0.15.7)

[warning] Too many branches (16 > 12)

(PLR0912)